<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>547</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>547</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>547</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Digitalização</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbUnidade">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="title">
     <string>Unidade</string>
    </property>
    <widget class="QRadioButton" name="rbParque">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
     </property>
     <property name="text">
      <string>VW - Parque</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rbCitroen">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Citroën</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rbVolkswagen">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Volkswagen</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rbJeep">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Jeep</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rbInvisible_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Invisible</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QComboBox" name="cbFormadePagamento">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Boleto Agrupado</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Boleto em Anexo</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbFormadePagamento">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>57</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Forma de Pagamento</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pteCte">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCte">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Documentos de CT-e</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFatura">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbPrimeiraNota">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="lbPrimeiraNota">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>6</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Primeira nota já foi inserida?</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbInvisible">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Supposed to be invisble</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rbSim">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Sim</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rbNao">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Não</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbProcurarFatura">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>239</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Procurar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbFatura">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="title">
     <string>Fatura</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDigitalizacao">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 0, 93);</string>
    </property>
    <property name="text">
     <string>Digitalização de CT-e</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDatadeVencimento">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="inputMask">
     <string>00/00/0000</string>
    </property>
    <property name="text">
     <string>//</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbDatadeVencimento">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Data de vencimento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbIniciar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLocalizacaoCte">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>299</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Procurar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbProcurarCte">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="title">
     <string>Localização dos documentos de CT-e</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLocalizacaoCte">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePrimeiroCte">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPrimeiroCte">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Nº do primeiro CT-e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLimpar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>470</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgGrupoSaga">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/gruposaga.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>gbProcurarCte</zorder>
   <zorder>gbFatura</zorder>
   <zorder>gbPrimeiraNota</zorder>
   <zorder>gbUnidade</zorder>
   <zorder>cbFormadePagamento</zorder>
   <zorder>lbFormadePagamento</zorder>
   <zorder>pteCte</zorder>
   <zorder>lbCte</zorder>
   <zorder>leFatura</zorder>
   <zorder>pbProcurarFatura</zorder>
   <zorder>lbDigitalizacao</zorder>
   <zorder>leDatadeVencimento</zorder>
   <zorder>lbDatadeVencimento</zorder>
   <zorder>pbIniciar</zorder>
   <zorder>pbLocalizacaoCte</zorder>
   <zorder>leLocalizacaoCte</zorder>
   <zorder>lePrimeiroCte</zorder>
   <zorder>lbPrimeiroCte</zorder>
   <zorder>pbLimpar</zorder>
   <zorder>imgGrupoSaga</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>415</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>rbCitroen</tabstop>
  <tabstop>rbJeep</tabstop>
  <tabstop>rbVolkswagen</tabstop>
  <tabstop>rbParque</tabstop>
  <tabstop>rbInvisible_2</tabstop>
  <tabstop>cbFormadePagamento</tabstop>
  <tabstop>rbSim</tabstop>
  <tabstop>rbNao</tabstop>
  <tabstop>rbInvisible</tabstop>
  <tabstop>lePrimeiroCte</tabstop>
  <tabstop>leDatadeVencimento</tabstop>
  <tabstop>pbProcurarFatura</tabstop>
  <tabstop>leFatura</tabstop>
  <tabstop>pbLocalizacaoCte</tabstop>
  <tabstop>leLocalizacaoCte</tabstop>
  <tabstop>pteCte</tabstop>
  <tabstop>pbIniciar</tabstop>
  <tabstop>pbLimpar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
